src/labels/CustomLabels.labels

```diff
@@ -5650,6 +5714,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Label for selecting fields to include as columns</shortDescription>
         <value>Select Fields to Display</value>
     </labels>
+    <labels>
+        <fullName>geHeaderPaymentServices</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>iFrame header for Payment Services information</shortDescription>
+        <value>Powered by Payment Services</value>
```

Comment:
```suggestion
        <value>Powered by {0}</value>
```

Reply:
@psadie-sfdo I think we were going to do a variable here and pull in the common label for Payment Services. That way, if (when) Payment Services changes to a different name, we can just update the common label and have this one match.